testcontainers/src/core/wait/mod.rs

```diff
@@ -44,6 +48,8 @@ pub enum WaitFor {
     Http(HttpWaitStrategy),
     /// Wait for the container to exit.
     Exit(ExitWaitStrategy),
+    /// Wait for a certain command to exit with a successful code.
+    SuccessfulCommand(CommandStrategy),
```

Comment:
I'd make it more general rather than specific.
We always can expose a constructor method like `successful_command`

```suggestion
    /// Wait for a certain command to exit with a successful code.
    Command(CommandStrategy),
```

Reply:
Thank you for the constructive feedback! I've made these changes.